README.md

```diff
@@ -4,11 +4,12 @@
 [![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_mp4_plugin/)
 [![CircleCI](https://circleci.com/gh/membraneframework/membrane_mp4_plugin.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_mp4_plugin)
 
-This plugin provides utilities for MP4 container parsing and serialization and elements for muxing the stream to [CMAF](https://www.wowza.com/blog/what-is-cmaf).
+This plugin provides utilities for MP4 container parsing and serialization along with elements for muxing the stream to MP4 or [CMAF](https://www.wowza.com/blog/what-is-cmaf).
```

Comment:
You should organize modules in the docs sidebar using groups, see https://hexdocs.pm/ex_doc/Mix.Tasks.Docs.html#module-groups

Reply:
I've made a comment inside `mix.exs`